File: SEMESTER 1/PROGRAMMING AND DATA STRUCTURES LAB/EXPERIMENT 2/QUESTION 2.c
#include <stdio.h>
 
int main()
{
     number, tempnum;
 
    printf("Enter an integer \n");
    
    
    scanf("%d", &number);
    
    
    tempnum = number;
    
    
        number = number << 2;
    printf("%d x 8 = %d\n", tempnum, number);
    
    }

File: SEMESTER 1/PYTHON PROGRAMMING LAB/EXPERIMENT 7/QUESTION 2.py
file=open(input("enter the file name "),"r")

#A) FIND THE MAX NUMBER

larg=0
for integer in file:
    if integer>larg:
        larg=integer
        
print("maximum number: ",larg)

B) FIND AVERAGE OF ALL NUMBERS


avg=0
count=0

for integer in file:
    avg+=integer
    count+=1
    
avg=avg/count

print("average of number: ",avg)

C) COUNT NUMBER OF NUMBERS GREATER THAN 100


count=0
for integer in file:
    if integer>100:
        count+=1

print("number of numbers greater than 100: ",count)
